ci/test-stable.sh

```diff
@@ -29,6 +29,9 @@ NPROC=$((NPROC>14 ? 14 : NPROC))
 echo "Executing $testName"
 case $testName in
 test-stable)
+  # start PostgreSQL database
+  /etc/init.d/postgresql start
```

Comment:
It's not safe to assume that the host machine will have postgresql installed.  For example it's common to run `ci/test-stable.sh` from your development machine.  The postgresql-dependent tests should just be ignored in this case

Reply:
Okay -- we have to check and ignore postgres setup failures and make the test_postgres_plugin to tolerate DB not setup.

Reply:
Is there a handy existing postgres env var we could use to indicate the presence of postgres maybe?